index.html

```diff
@@ -5,8 +5,49 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link rel="stylesheet" href="styles/index.css">
 </head>
-<body>
+<body >
+
+    <header>
+        <h1 id="title">Weather Report</h1>
+        <h2>for the lovely city</h2>
```

Comment:
You generally want to have your heading tags (`h1`, `h2`, etc) set up to build a table of contents (for accessibility reasons)  Instead of using a `h2` for the second part you should instead use a `p` or `span` with a class on it:

```suggestion
        <h1 id="title">Weather Report</h1>
        <p class="subtitle">for the lovely city</p>
```